diff -crB intro.js-2.0.orig/intro.js intro.js-2.0/intro.js
*** intro.js-2.0.orig/intro.js	2016-02-19 15:30:37.965214248 -0500
--- intro.js-2.0/intro.js	2016-02-19 15:39:47.247937992 -0500
***************
*** 870,876 ****
          var innerLi    = document.createElement('li');
          var anchorLink = document.createElement('a');
  
!         anchorLink.onclick = function() {
            self.goToStep(this.getAttribute('data-stepnumber'));
          };
  
--- 870,877 ----
          var innerLi    = document.createElement('li');
          var anchorLink = document.createElement('a');
  
!         anchorLink.onclick = function(event) {
!           event.stopPropagation ();
            self.goToStep(this.getAttribute('data-stepnumber'));
          };
  
***************
*** 921,927 ****
        //next button
        var nextTooltipButton = document.createElement('a');
  
!       nextTooltipButton.onclick = function() {
          if (self._introItems.length - 1 != self._currentStep) {
            _nextStep.call(self);
          }
--- 922,929 ----
        //next button
        var nextTooltipButton = document.createElement('a');
  
!       nextTooltipButton.onclick = function(event) {
!         event.stopPropagation ();
          if (self._introItems.length - 1 != self._currentStep) {
            _nextStep.call(self);
          }
***************
*** 933,939 ****
        //previous button
        var prevTooltipButton = document.createElement('a');
  
!       prevTooltipButton.onclick = function() {
          if (self._currentStep != 0) {
            _previousStep.call(self);
          }
--- 935,942 ----
        //previous button
        var prevTooltipButton = document.createElement('a');
  
!       prevTooltipButton.onclick = function(event) {
!         event.stopPropagation ();
          if (self._currentStep != 0) {
            _previousStep.call(self);
          }
***************
*** 948,954 ****
        skipTooltipButton.href = 'javascript:void(0);';
        skipTooltipButton.innerHTML = this._options.skipLabel;
  
!       skipTooltipButton.onclick = function() {
          if (self._introItems.length - 1 == self._currentStep && typeof (self._introCompleteCallback) === 'function') {
            self._introCompleteCallback.call(self);
          }
--- 951,958 ----
        skipTooltipButton.href = 'javascript:void(0);';
        skipTooltipButton.innerHTML = this._options.skipLabel;
  
!       skipTooltipButton.onclick = function(event) {
!         event.stopPropagation ();
          if (self._introItems.length - 1 == self._currentStep && typeof (self._introCompleteCallback) === 'function') {
            self._introCompleteCallback.call(self);
          }
***************
*** 1323,1329 ****
          // when user clicks on the hint element
          hint.onclick = function(e) {
            var evt = e ? e : window.event;
!           if (evt.stopPropagation)    evt.stopPropagation();
            if (evt.cancelBubble != null) evt.cancelBubble = true;
  
            _hintClick.call(self, hint, item, i);
--- 1327,1333 ----
          // when user clicks on the hint element
          hint.onclick = function(e) {
            var evt = e ? e : window.event;
!           evt.stopPropagation();
            if (evt.cancelBubble != null) evt.cancelBubble = true;
  
            _hintClick.call(self, hint, item, i);
***************
*** 1458,1464 ****
      var closeButton = document.createElement('a');
      closeButton.className = 'introjs-button';
      closeButton.innerHTML = this._options.hintButtonLabel;
!     closeButton.onclick = _hideHint.bind(this, stepId);
  
      tooltipTextLayer.appendChild(tooltipWrapper);
      tooltipTextLayer.appendChild(closeButton);
--- 1462,1472 ----
      var closeButton = document.createElement('a');
      closeButton.className = 'introjs-button';
      closeButton.innerHTML = this._options.hintButtonLabel;
!     var self = this;
!     closeButton.onclick = function (event) {
!       event.stopPropagation ();
!       _hideHint.bind(self, stepId);
!     }
  
      tooltipTextLayer.appendChild(tooltipWrapper);
      tooltipTextLayer.appendChild(closeButton);
Binary files intro.js-2.0.orig/.intro.js.swp and intro.js-2.0/.intro.js.swp differ
